### PR TITLE
Generalized gitfs config, cleaned up ssl setup

### PR DESCRIPTION
**Pull request by blarghmatey**
- Moved explicit gitfs config setup to just use pillar driven master
  config
- Fixed variable use in ssl setup to fix errors

### CODE REVIEW

File: master/templates/conf-template.conf [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-  {% for arg, val in config.params.items() %}
-  - {{ arg }}: {{ val }}
-  {% endfor %}
-{% else %}
-{% for arg, val in config.items() %}
-{{ arg }}: {{ val }}
-{% endfor %}
-{% endif %}
\ No newline at end of file
+{%- macro render_yaml(config_dict, offset=2) -%}
```
Comment (bdero) [inline master/templates/conf-template.conf]: Is this macro defined within our repos somewhere? I looked for it in the docs but couldn't seem to find it: https://docs.saltstack.com/en/develop/ref/renderers/all/salt.renderers.jinja.html

Comment (blarghmatey) [inline master/templates/conf-template.conf]: That's the opening for the definition of the macro.

https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#jinja-macros

Comment (bdero) [inline master/templates/conf-template.conf]: Ah, right.

**Comment by bdero** [comment]
:+1:

**Pull request merged**